src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/LocationService.kt

```diff
@@ -364,7 +364,7 @@ class LocationService(
       null,
     )
 
-    return locationToUpdate.toDto(includeParent = true)
+    return locationToUpdate.toDto(includeParent = true, includeChildren = reactivateSubLocations)
```

Comment:
If you cascade the re-activation you need to send events for reactivation of sub locations